### ISSUE TITLE
Results not the same while using pretrained model.

### ISSUE THREAD

**Issue by SonDaoDuy** [ehsanik/SeGAN#4]
Hello,
I have run the code with the pretrained model you provided. However, It cannot produce the same results as yours.
These are the results running on the test set, using SV gt:
![segan_res](https://user-images.githubusercontent.com/9215877/58409655-1a40f500-80a3-11e9-98a1-d52396594a9b.PNG)
These are the results running on the val set (quite close to yours), using SV gt:
![segan_res_val](https://user-images.githubusercontent.com/9215877/58409750-4e1c1a80-80a3-11e9-9f3d-83cf63e7025a.PNG)

Is there any problem with the pretrained model? Do you have other pretrained that produce the same results as yours on test set?

Also in your paper, I cannot find how many epochs and batch size you use to train the model. Therefore, I just use the default parameters on your SettingsParser.lua with batch size changed to 16 to train the model from scratch. Is it correct to do that?

**Comment by DrewdropLife** [comment]
Hello, I've been studying this model lately. When you start training this network from scratch, what is the size of your generated network? Will the image be all black during the test? The size of the generated network that I trained from scratch is only 435M; I also used the pretrained model and continued to train on a small part of the author's dataset, the size of the model also changed from 560M to 435M, and the output results were all black. Do you encounter this problem?

**Comment by SonDaoDuy** [comment]
It was a mistake. I changed the category to 'chair', but forgot to change back when testing.
@changxinC I'm not sure. Let me check it later.

**Comment by ehsanik** [comment]
@changxinC Can you give me some details on which command are you using? The image should not be all black during test. Are you sure you are normalizing the images back to 0-255 before visualizing them?

**Comment by DrewdropLife** [comment]
@ehsanik Hello, I have opened a new question, thank you for your help!
